src/test/java/br/com/alura/comex/repository/PedidoRepositoryTests.java

```diff
@@ -0,0 +1,49 @@
+package br.com.alura.comex.repository;
+
+
+import br.com.alura.comex.controller.dto.RelatorioPedidosPorCategoriaProjection;
+import org.junit.Test;
```

Comment:
Aqui, seria a anotação do JUnit 5.

```suggestion
import org.junit.jupiter.api.Test;
```